Tasks/QuickPerfTest/Invoke-QuickPerfTest.ps1

```diff
@@ -226,6 +240,12 @@ $CltAccountUrl = ComposeAccountUrl($VSOAccountUrl)
 
 Write-Verbose "VSO account Url = $VSOAccountUrl" -Verbose
 
+$tfsUrl = $env:System_TeamFoundationCollectionUri.TrimEnd('/')
+
+$resultsMDFolder = New-Item -ItemType Directory -Force -Path "$env:Temp\LoadTestResultSummary"
+Remove-Item $resultsMDFolder\QuickPerfTestResults_*.md -Force
+$summaryFile =  ("{0}\QuickPerfTestResults_{1}_{2}.md" -f $resultsMDFolder, $env:SYSTEM_DEFINITIONID, $env:BUILD_BUILDID)
```

Comment:
>QuickPerfTestResults_ [](start = 22, length = 21)

As a safety mechanism shall we put task specific string and datetime string as well?